src/translations/english.json

```diff
@@ -107,6 +114,7 @@
     "accreditationsAdaptationFund" : "Adaptation Fund",
     "aboutGreenClimate"            : "<p>On February 27, 2019, &quot;Environmental Project Implementation Unit&quot; State Agency became the first national entity among 14 countries of Eastern Europe, Central Asia and the Caucasus region accredited by the Board of the Green Climate Fund. </p>\n<p><br></p><p>Accreditation provides EPIU with direct access to the GCF's funds, making it possible to implement grant projects worth up to 10 million US dollars.</p>\n<p><br></p>\n",
     "aboutAdaptionFund"            : "<p>On November 4, 2016, &quot;Environmental Project Implementation Unit&quot; State Agency was accredited to the Adaptation Fund of the UN Framework Convention on Climate Change.</p>\n<p><br></p> <p>On October 12, 2018, the Board of the Adaptation Found approved the &quot;Artik city closed stonepit wastes and flood management&quot; pilot grant project with funding in the amount of 1 435 100 US dollars. </p>\n<p><br></p>\n <p>On March 23, 2019, the grant project 'Strengthening land based adaptation capacity in communities adjacent to protected areas in Armenia' (US$ 2 506 000) was approved.</p>\n<p><br></p>",
+    "aboutAdaptionEpiu"            : "<p>On March 1, 2023, EPIU was re-accredited to the Adaptation Fund by the decision of the AF Board.</p>",
```

Comment:
This Should not be separate property, this text should be inside the text above, in the property 
`aboutAdaptionFund`

Same for Russian and Armenian languages.